dev-libs/keystone/keystone-0.9.2-r1.ebuild

```diff
@@ -42,6 +42,12 @@ REQUIRED_USE="
 	python? ( ${PYTHON_REQUIRED_USE} )
 "
 
+# Upstream doesn't provide automated tests
+RESTRICT="!test? ( test )"
```

Comment:
Normally we use this `!test? ( test )` construction to enable tests only if the test use flag is set (e.g. the tests require extra dependencies).

Since there are no tests even if we set `USE=test`, we need to change this to unconditionally disable tests

```suggestion
# Upstream doesn't provide automated tests
RESTRICT="test"
```